src/modules/visitor-space/components/FilterMenu/FilterMenu.module.scss

```diff
@@ -169,4 +169,16 @@ $checkbox-size: $spacer-sm;
 	&__overlay {
 		z-index: get-z-layer("filter-overlay");
 	}
+
+	:global(.c-dropdown__trigger:first-child + .c-dropdown__content-open) {
+		:global(.c-menu) {
+			margin-top: -1px;
+			padding: 0;
+			margin-left: $spacer-sm;
+
+			div > ul > li > div > div > button {
+				padding-left: $spacer-sm !important;
+			}
```

Comment:
You don't need this anymore, it should already add a margin by setting margin-left on line 177
```suggestion
```